README.md

```diff
@@ -106,6 +106,52 @@ to allocate more memory for its executions than what's allowed.
 you can also use the [rate limiter](https://github.com/triton-inference-server/server/blob/main/docs/rate_limiter.md)
 in Triton to limit the number of requests allowed to enter execution.
 
+## Auto-Complete Model Configuration
+
+Assuming Triton was not started with `--disable-auto-complete-config` command line
+option, the Tensorflow Backend makes use of the metadata available in TensorFlow
+SavedModel to populate the required fields in the model's config.pbtxt. You can
+learn more about Triton's support for auto-completing model configuration from
+[here](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#auto-generated-model-configuration).
+
+However, in Graphdef format, models do not carry sufficient metadata and hence
+Triton cannot generate model configuration for them. As a result, config.pbtxt
+must be provided for such models explicitly.
+
+Tensorflow backend can complete the following fields in model configuration:
+
+### max_batch_size
+
+Auto-completing max_batch_size follows the following rules:
+
+1. Autocomplete has determined the model is capable of batching requests.
+2. max_batch_size is 0 in the model configuration or max_batch_size
+   is omitted from the model configuration.
+
+If the above two rules are met, max_batch_size is set to
+[default-max-batch-size](#--backend-config=tensorflow,default-max-batch-size=\<int\>).
+Otherwise max_batch_size is set as 0.
```

Comment:
I think it would be great if we can mention the default max batch size.

Reply:
It is mention below if you just follow the link. I wanted to avoid the duplicity.